src/map.cpp

```diff
@@ -4495,6 +4495,9 @@ void map::process_items_in_vehicle( vehicle &cur_veh, submap &current_submap, co
                 it_temp = std::min( it_temp, temperatures::fridge );
             } else if( pt.enabled && pti.has_flag( VPFLAG_FREEZER ) ) {
                 it_temp = std::min( it_temp, temperatures::freezer );
+            } else if( pt.enabled && pti.has_flag( VPFLAG_HEATER ) ) {
+                it_temp = std::min( it_temp, temperatures::heater );
```

Comment:
Here you probably want to use a std::max since it's heating up and it limited by the outside temperature (i.e. cannot heat hotter than outside temp, would mean it can cool things in hot weather).

E.g. 1 outside temp is 35, heater temp is 140.  This will set it_temp to 35, but you want it to be 140.

E.g. 2 *somehow* outside temp is I dunno 150?  and heater is 140, you want max, which is 150.

The likelihood of outside actually being hotter than the heater I found unlikely but doing an std::max anyway doesn't hurt and is future proof.

Other than that I think things seem good from what I can tell.

Reply:
> The likelihood of outside actually being hotter than the heater I found unlikely but doing an std::max anyway doesn't hurt and is future proof.

I want my frozen food hot so quick i'm willing to drive over a lava fault?

Reply:
Yeah I mean in the current game this is highly unlikely but I what if, and this is just an example I pulled out of thin-air, a heat wave or something was added in the future that made scorching temperatures?  I dunno, like I said, it's probably not totally necessary but it's worth doing.

Reply:
Y'know, now that you mention it, scorching heat-flashes caused by the Cataclysm would be a neat rare weather effect.